src/commands/restriction/account.ts

```diff
@@ -72,9 +81,20 @@ export default class extends ProfileCommand {
     execute(options: CommandOptions) {
         this.spinner.start();
         const profile = this.getProfile(options);
+
+        const password = options.password || readlineSync.question('Enter your wallet password: ');
+        new PasswordValidator().validate(password);
+        const passwordObject = new Password(password);
+
+        if (!profile.isPasswordValid(passwordObject)) {
```

Comment:
Is #89 validation necessary if we have ``new PasswordValidator().validate(password);`` in the line #86? (applies to other commands)

Reply:
in the line #86 it validates that the SDK accepts the password (8 char min)
profile.isPasswordValid validates that the password matches the Simple Wallet
They don't throw the same error
I'll check if something better is doable, but I think that's acceptable like that